google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/RemoveEntireSitelinkCampaignExtensionSetting.java

```diff
@@ -0,0 +1,192 @@
+    }
+
+    try {
+      new RemoveEntireSitelinkCampaignExtensionSetting()
+          .runExample(googleAdsClient, params.customerId, params.campaignId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+      System.exit(1);
+    }
+  }
+
+  /** Runs the example. */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates an operation to remove a campaign extension setting.
+    CampaignExtensionSettingOperation extensionSettingOperation =
+        CampaignExtensionSettingOperation.newBuilder()
+            .setRemove(
+                ResourceNames.campaignExtensionSetting(
+                    customerId, campaignId, ExtensionType.SITELINK))
+            .build();
+
+    // Retrieves all sitelink extension feed items for a customer and campaign.
+    List<String> extensionFeedItems =
+        getSitelinkExtensionFeedItems(googleAdsClient, customerId, campaignId);
+
+    // Constructs operations to remove the extension feed item operations.
+    List<MutateOperation> extensionFeedItemOperations =
+        extensionFeedItems.stream()
+            .map(
+                sitelinkExtension ->
+                    CampaignExtensionSettingOperation.newBuilder()
+                        .setRemove(sitelinkExtension)
+                        .build())
+            .map(
+                extensionOperation ->
+                    MutateOperation.newBuilder()
+                        .setCampaignExtensionSettingOperation(extensionOperation)
+                        .build())
+            .collect(Collectors.toList());
+
+    // Creates a list of operations that contains both the campaign extension setting and the feed
+    // item operations.
+    List<MutateOperation> allOperations = new ArrayList();
+    allOperations.add(
+        MutateOperation.newBuilder()
+            .setCampaignExtensionSettingOperation(extensionSettingOperation)
+            .build());
+    allOperations.addAll(extensionFeedItemOperations);
+
+    // Connects to the API.
+    try (GoogleAdsServiceClient client =
+        googleAdsClient.getLatestVersion().createGoogleAdsServiceClient()) {
+      // Issues the request.
+      MutateGoogleAdsResponse response = client.mutate(String.valueOf(customerId), allOperations);
+
+      // Prints the result of removing the campaign extension setting.
+      System.out.printf(
```

Comment:
Please also include [this explanation](https://github.com/googleads/google-ads-php/blob/943c01fb48bcc1823e84a7b99ba6fdc507bd041d/examples/Extensions/RemoveEntireSitelinkCampaignExtensionSetting.php#L147).